homemate/src/pages/profile/index.js

```diff
@@ -140,6 +144,65 @@ function Profile() {
     return '';
   };
 
+  function open() {
+    window.location.href = userData?.socialMedia;
```

Comment:
```suggestion
    window.open(`${userData?.socialMedia}`, '_blank')
```

Abre a rede social em uma nova aba. acho que fica mais interessante.